config/samples/clusterinstance-defaults-configmap.yaml

```diff
@@ -0,0 +1,68 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: clustertemplate-sample.v1.0.0-clusterinstance-defaults
+  namespace: clustertemplate-a-v4-16
+data:
+  clusterProvisioningTimeout: "80" # optional
```

Comment:
We should probably be using https://pkg.go.dev/time#Duration for this (e.g. 80m, 1h, 30s) and all the other timeout values

Reply:
Good point about using time duration format for these config values. I think that looks better than just using a number.